code/modules/mob/living/silicon/robot/drone/drone.dm

```diff
@@ -32,12 +32,12 @@ var/list/mob_hat_cache = list()
 	pass_flags = PASS_FLAG_TABLE
 	braintype = "Drone"
 	lawupdate = FALSE
-	density = TRUE
+	density = FALSE
 	req_access = list(ACCESS_ENGINEERING_LVL2, ACCESS_ROBOTICS)
 	integrated_light_max_bright = 0.5
 	local_transmit = 1
 	possession_candidate = 1
-	speed = -2
```

Comment:
why

Reply:
that's what the Tegu code did, I think it makes the drones faster?

Reply:
It does make them faster, but why should they be faster?

Reply:
"Allows AI to do more stuff, simple as that.
    Drones used to be too fast and could block people(especially annoying). This isn't the case anymore.
    Just QOL."